build.sbt

```diff
@@ -46,34 +46,40 @@ ThisBuild / scalaVersion := Scala213
 ThisBuild / githubWorkflowPublishTargetBranches := Seq() // disable publication for now
 
 ThisBuild / githubWorkflowBuildMatrixAdditions +=
-  "platform" -> List("jvm", "js")
+  "platform" -> List("jvm", "js", "native")
 
 ThisBuild / githubWorkflowBuildMatrixExclusions ++=
-  githubWorkflowJavaVersions.value.filterNot(Set(PrimaryJava)).map { java =>
-    MatrixExclude(Map("platform" -> "js", "java" -> java))
+  githubWorkflowJavaVersions.value.filterNot(Set(PrimaryJava)).flatMap { java =>
+    Seq(MatrixExclude(Map("platform" -> "js", "java" -> java)),
+        MatrixExclude(Map("platform" -> "native", "java" -> java))
+    )
   }
 
+ThisBuild / githubWorkflowBuildMatrixExclusions ++= Seq(DottyOld, DottyNew).map { dottyVersion =>
+  MatrixExclude(Map("platform" -> "native", "scala" -> dottyVersion))
+} // Dotty is not yet supported by Scala Native
+
 // we don't need this since we aren't publishing
 ThisBuild / githubWorkflowArtifactUpload := false
 
+ThisBuild / githubWorkflowBuildMatrixFailFast := Some(false)
+
 val JvmCond = s"matrix.platform == 'jvm'"
 val JsCond = s"matrix.platform == 'js'"
+val NativeCond = s"matrix.platform == 'native'"
 
 val Scala2Cond = s"(matrix.scala != '$DottyOld' && matrix.scala != '$DottyNew')"
 val Scala3Cond = s"(matrix.scala == '$DottyOld' || matrix.scala == '$DottyNew')"
```

Comment:
We don't need this anymore, do we? I wonder why this split was in here in the first place; maybe @djspiewak can tell.

Reply:
> We don't need this anymore, do we? I wonder why this split was in here in the first place; maybe @djspiewak can tell.

It isn't used anywhere anymore. Can remove.

Reply:
Actually just looking at the build again there are a couple of spots `Scala2Cond` is used as a condition. `Scala3Cond`, not anymore.

Reply:
Reverted to original setup. Can make a separate pull request for build specific changes.